src/augtool.c

```diff
@@ -481,7 +534,14 @@ static int main_loop(void) {
 
     while(1) {
         if (get_line) {
```

Comment:
What's the value to the user that they can type `augtool -l` rather than `auglua` to get the lua-based shell ? With the latter, `auglua` would become a separate executable.